src/web/src/components/CourseListing.vue

```diff
@@ -11,15 +11,14 @@
         <button
           v-if="actions.add || actions.remove"
           class="btn"
-          @click="toggleCourse(course)"
+          @click="actions.remove ? toggleCourse(course):null"
         >
           <font-awesome-icon v-if="course.selected" :icon="faTimes" />
           <font-awesome-icon v-else :icon="faPlus" />
         </button>
         <button
           v-if="actions.collapse"
           class="btn"
-          @click="toggleShowSection()"
```

Comment:
er where did this go

Reply:
It happens when you click anywhere in the box, having this there will double click the event

Reply:
yes, this works right now because we aren't using the collapse element in `CourseList`. but if we added added `actions.collapse` to `CourseList` it wouldn't work.
if you add `@click.stop`, when you click on the button it won't pass the click to the box

Reply:
So the beginning would be click.stop, then add the click at the bottom?

Reply:
If we added a drop down here, we'd have to decide what the default behavior of clicking the box would do, which would require a logic change here anyway

Reply:
so `@click.stop="toggleShowSection()"` and same for the other click

Reply:
So it stops the other click and does the new one?